apps/website/src/pages/docs/avatar.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Avatar
-description: A circular identifier that displays the user’s custom set photo or the user’s first and last name initials.
+description: A circular identification element that prominently showcases the user's customized profile picture or their first and last name initials.
 layout: ./_layout.astro
 propsPath: '@itwin/itwinui-react/esm/core/Avatar/Avatar.d.ts'
```

Comment:
we should show props for both Avatar and AvatarGroup

h2: Props
  h3: Avatar
  h3: AvatarGroup

Reply:
frontmatter variable can be removed 
![image](https://github.com/iTwin/iTwinUI/assets/9084735/e08631f0-d7ca-4cac-9165-6ea34eba8bda)